examples/github_actions_orchestration/run.py

```diff
@@ -0,0 +1,63 @@
+#  Copyright (c) ZenML GmbH 2022. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+import random
```

Comment:
If you still have the time, it would be cool to put this into our best practices directly, feel free to copy it from the vertex ai PR - its the exact same pipeline, all you'll need to to is change the pipeline name

Reply:
I think copying a directory is not that much more complicated and it will make it easier to go from example to project of your own. So I would eventually move to the directory structure, up to you though if it should be done within this PR still